File: washwise-backend/src/main/java/com/washwise/dto/response/ServiceResponse.java
package com.washwise.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ServiceResponse {

    private UUID id;
    private String name;
    private String description;
    private BigDecimal price;
    private String category;
    private String duration;
    private Boolean isActive;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: washwise-mobile/app/src/main/java/com/washwise/mobile/feature/service/data/ServiceResponse.kt
package com.washwise.mobile.feature.service.data

data class ServiceResponse(
    val id: String,
    val name: String,
    val description: String?,
    val category: String?,
    val price: Double,
    val duration: String?,
    val imageUrl: String?,
    val isActive: Boolean?
)
